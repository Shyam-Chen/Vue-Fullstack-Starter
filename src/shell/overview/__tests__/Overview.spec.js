import { shallowMount } from '@vue/test-utils';

import Overview from '../Overview';

describe('Overview', () => {
  let [wrapper] = [];

  beforeEach(() => {
    wrapper = shallowMount(Overview);
  });

  it('should render an initial component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
