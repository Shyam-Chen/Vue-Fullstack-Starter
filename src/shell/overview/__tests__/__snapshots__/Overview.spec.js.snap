// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview should render an initial component 1`] = `
<v-container fluid="" class="shell-container">
  <v-layout column="">
    <div class="headline">Practical Examples</div>
    <v-layout column="" class="mt-1">
      <div>00. Hello World - A bare-bones Hello World example.</div>
      <div>01. Guide - Build modern web applications using the Vue stack.</div>
      <div>02. CRUD Operations - The four major functions implemented in applications.</div>
      <div>03. Controls - Get or collect information from users for applications.</div>
      <div>04. Data Table - Process and display of data from applications.</div>
      <div>05. Internationalization - Adapt applications and text to enable their usability in a particular cultural or linguistic market.</div>
      <div>06. Data Visualization - Produce dynamic, interactive data visualizations in applications.</div>
      <div>07. Authorization - Provide a user identity to applications.</div>
      <div>08. Payment - Process payments from users for applications.</div>
      <div>09. Messages - Communicate or deliver messages to users for applications.</div>
      <div>10. Machine Learning - Improve user experience through artificial intelligence.</div>
    </v-layout>
  </v-layout>
</v-container>
`;
